tools/license-builder.sh

```diff
@@ -77,6 +77,8 @@ licenseText="$(sed -e '/You should have received a copy of the CC0/,$d' -e 's/^\
 addlicense "SipHash" "deps/v8/src/third_party/siphash" "$licenseText"
 licenseText="$(sed -e '/The data format used by the zlib library/,$d' -e 's/^\/\* *//' -e 's/^ *//' "${rootdir}/deps/zlib/zlib.h")"
 addlicense "zlib" "deps/zlib" "$licenseText"
+addlicense "simdutf" "deps/simdutf" "$licenseText"
+licenseText="$(curl -sL https://raw.githubusercontent.com/simdutf/simdutf/HEAD/LICENSE-MIT)"
```

Comment:
```suggestion
licenseText="$(curl -sL https://raw.githubusercontent.com/simdutf/simdutf/HEAD/LICENSE-MIT)"
addlicense "simdutf" "deps/simdutf" "$licenseText"
```

Reply:
Thanks for the review. Updated my pull request.